core/envelope-sim/src/main/kotlin/fr/sncf/osrd/envelope_sim/etcs/Constants.kt

```diff
@@ -13,6 +13,12 @@ package fr.sncf.osrd.envelope_sim.etcs
  */
 const val qNvinhsmicperm = false
 
+/**
+ * National Default Value: permission to follow release speed at 40km/h near the EoA. See Subset
+ * 026: table in Appendix A.3.2.
+ */
+const val releaseSpeed = 40.0 / 3.6 // m/s
```

Comment:
Separate matter (as the release speed is probably actually the min of a lot of values, we may avoid name-squating it):
```suggestion
const val nationalReleaseSpeed = 40.0 / 3.6 // m/s
```